LKMEX-whitelist/HodlTokenClub WL.md

```diff
@@ -0,0 +1,91 @@
+those who bring valuable use cases (for example, use cases that go against the principles outlined above, where LKMEX 
+rules are overridden, re-enabling selling or buying will not be eligible). After the xExchange launch, 
+pre-whitelisted projects and SCs will potentially be revisited through a governance call.
+
+The project has to make a pull request in the current repository, where it adds a new file in the LKMEX-whitelist 
+folder by filling in the existing template document. The name of the document should be the name of the project.
+
+Note: Projects will lose the transfer role after upgrading the contract. You need to make a new pull request, where 
+new code updates brought to the smart contract must be explained and motivated.
+
+Projects can also lose their `transferRole` status if the Smart Contract is being used in ways that go against LKMEX 
+principles. Questionable activities will be reviewed by the MultiversX team and any Project or Smart Contract found in 
+breach of the rules will automatically lose their transferRole status.
+
+Whitelisted projects will be given a transferRole to the WXMEX tokenID, which is a token that does not 
+have energy and is used only by whitelisted projects. 
+
+The whitelisted projects will need to wrap the XMEX they want to deposit or use. By wrapping it, the XMEX will 
+temporarily “lose” its energy but the energy can be recovered and regained by unWrap it back.
+
+Projects will be able to use the wrappedXMEX to deposit and use in the whitelisted distribution Smart Contracts. Users who will
+get/claim wrappedXMEX will need to unWrap it in order to gain its original energy and convert it back to XMEX 
+in order to enter liquidity or farms.
+
+wrappedXMEX does not have the benefits of the XMEX. Only after unwrapping you gain all the benefits.
+
+### Usage specs
+
+- rewards_depositer_account - user account from project side that should deposit rewards in the project's distribution smart contract; this should NOT be a smart contract
+- distribution_sc - project's distribution smart contract that will receive transfer role for wrappedXMEX
+- xmex_wrapper_sc - xmex wrapper smart contract owned by xExchange owner entity
+- user_account - end user account that claims and uses the distributed rewards
+
+```mermaid
+sequenceDiagram
+    actor rewards_depositer_account
+    participant distribution_sc
+    participant xmex_wrapper_sc
+    actor user_account
+    rewards_depositer_account->>xmex_wrapper_sc: wrapLockedToken XMEX
+    xmex_wrapper_sc-->>rewards_depositer_account: remove energy & transfer wXMEX
+    rewards_depositer_account->>distribution_sc: deposit wXMEX
+    user_account->>distribution_sc: claim wXMEX
+    rect rgba(191, 223, 255, .1)
+    note right of xmex_wrapper_sc: Integrated in xExchange
+    user_account->>xmex_wrapper_sc: unwrapLockedToken wXMEX
+    xmex_wrapper_sc-->>user_account: add energy & transfer XMEX
+    end
+```
+
```

Comment:
you can delete all the lines until here.

Reply:
Done. all requested lines deleted @sasurobert